server/services/src/main/java/org/zanata/action/VersionHomeAction.java

```diff
@@ -1402,4 +1416,8 @@ public CopyVersionHandler getCopyVersionHandler() {
     public AbstractListFilter<HLocale> getDocumentsTabLanguageFilter() {
         return this.documentsTabLanguageFilter;
     }
+
+    public boolean isMTEnabled() {
+        return identity.hasRole("mt-bulk") && mtServiceURL != null;
```

Comment:
Should the order be reversed? (Not blocking, just a thought)

Reply:
Yup. Done